core/src/main/java/com/sequenceiq/cloudbreak/service/externaldatabase/AwsDatabaseServerParameterSetter.java

```diff
@@ -0,0 +1,33 @@
+package com.sequenceiq.cloudbreak.service.externaldatabase;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.cloudbreak.common.mappable.CloudPlatform;
+import com.sequenceiq.cloudbreak.service.externaldatabase.model.DatabaseServerParameter;
+import com.sequenceiq.redbeams.api.endpoint.v4.stacks.DatabaseServerV4StackRequest;
+import com.sequenceiq.redbeams.api.endpoint.v4.stacks.aws.AwsDatabaseServerV4Parameters;
+
+@Component
+public class AwsDatabaseServerParameterSetter implements DatabaseServerParameterSetter {
+
+    @Value("${cb.aws.externaldatabase.retentionperiod:1}")
+    private int retentionPeriod;
+
+    @Value("${cb.aws.externaldatabase.engineversion:10.6}")
+    private String engineVersion;
+
+    @Override
+    public void setParameters(DatabaseServerV4StackRequest request, DatabaseServerParameter serverParameter) {
```

Comment:
`DatabaseServerParameter serverParameter` parameter is not used. Are you sure this is the right method signature?

Reply:
it is used here.